.storybook/modes.ts

```diff
@@ -0,0 +1,13 @@
+export const allModes = {
+    large: {
+        viewport: 'l',
+    },
+    // Note, you can still specify the more
+    // specific options listed in the section above
```

Comment:
what does "section above" refer to?

Reply:
it looks like this file/object is used once, what about adding `viewport` property directly to preview.tsx?